### PR TITLE
Remove the log manager dependencies and explicitly exclude jcl-over-s…

### PR DESCRIPTION
**Pull request by jamezp**
…lf4j

Just let me know if this needs a JIRA.

### CODE REVIEW

**Comment by luck3y** [comment]
this is ok to test

File: client/jaxws-client/pom.xml [hunk 1]
```diff
@@ -96,15 +96,15 @@
...
-                                <dependency>
-                                    <groupId>org.jboss.slf4j</groupId>
-                                    <artifactId>slf4j-jboss-logmanager</artifactId>
-                                </dependency>
-                                <dependency>
-                                    <groupId>org.jboss.logmanager</groupId>
-                                    <artifactId>jboss-logmanager</artifactId>
-                                </dependency>
                             </includeDependencies>
+                            <addExclusions>
```
Comment (emmartins) [inline client/jaxws-client/pom.xml]: this not needed, master already excludes jcl-over-slf4j

File: client/jaxws-client/pom.xml [hunk 2]
```diff
@@ -96,15 +96,15 @@
                                     <groupId>org.jboss.spec.javax.annotation</groupId>
                                     <artifactId>jboss-annotations-api_1.3_spec</artifactId>
                                 </dependency>
-                                <dependency>
```
Comment (emmartins) [inline client/jaxws-client/pom.xml]: without these deps the service module of quickstart jaxws-retail throws error complaining that slf4j implies is missing:
```
[ERROR] SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[ERROR] SLF4J: Defaulting to no-operation (NOP) logger implementation
[ERROR] SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

IIRC sl4fj-jboss-logmanager was added to the BOM cause it includes slf4j impl classes.

Comment (jamezp) [reply client/jaxws-client/pom.xml]: I'm not sure what to do here. We can't make assumptions about the users log manager. One possibility is to use the `org.jboss.logging:slf4j-jboss-logging`. However this has not been updated in some time.

File: client/jaxws-client/pom.xml (hunk 1 above)
Comment (jamezp) [reply client/jaxws-client/pom.xml]: Ah this must have been recently done. I'll update the PR.

**Comment by jamezp** [comment]
This has been open for almost a year so I'm guessing it's not desired.

**Comment by emmartins** [comment]
You did not reply to my comment ^^

**Comment by emmartins** [comment]
@jamezp I am open for a refresh/rework of the managed logging deps, and no one better to do that than you to indicate what the user apps projects should depend on, but of course quickstarts such as jaws-retail need to work properly (i.e. no logging error) so maybe some changes on quickstarts need to be done too?

**Comment by jamezp** [comment]
Well somehow I missed that :) I'd have to have another look at this as it may need some other changes at this point. Everything is working now though.

The main thing is that we shouldn't be using our implementations of logging bridges in client applications by default. The user should be able to decide that.

**Pull request closed without merging**